integration/kubernetes/init.sh

```diff
@@ -40,6 +40,33 @@ wait_pods_ready()
 	done
 }
 
+build_custom_stress_image()
+{
+	info "Build custom stress image"
+	image_version=$(get_test_version "docker_images.registry.version")
+	registry_image="docker.io/library/registry:${image_version}"
```

Comment:
can you get this URL from the versions.yaml file?

Reply:
@devimc , we can get the image name from the URl in versions.yaml  as

`registry_image=$(get_test_version "docker_images.registry.url" | cut -d '/' -f 5-):"${image_version}"`

I updated it, can you PTAL?